app/api/menu_routes.py

```diff
@@ -0,0 +1,61 @@
+from flask import Blueprint, request, jsonify
+from app.models import db, Menu, Business
+from ..forms.menu_form import MenuForm
+from flask_login import current_user, login_required
+from .auth_routes import validation_errors_to_error_messages
+
+menu_routes = Blueprint("menus", __name__, url_prefix="/menus")
+
+#Get All Menus
+@menu_routes.route("")
+def all_menus():
+    menus = Menu.query.all()
+    return {"menus": [m.to_dict() for m in menus]}
+
+#Create Menu
+@menu_routes.route("/", methods=["POST"])
+@login_required
+def create_menu():
+    form = MenuForm()
+    form["csrf_token"].data = request.cookies["csrf_token"]
```

Comment:
Authorized user must be owner of the business to create a menu for a business. 

Should be moved to business routes.